test-suite/shared/src/test/scala/org/scalajs/testsuite/javalib/util/LinkedHashMapTest.scala

```diff
@@ -39,17 +44,13 @@ abstract class LinkedHashMapTest extends HashMapTest {
     val lhm = factory.empty[Int, String]
     (0 until 100).foreach(key => lhm.put(key, s"elem $key"))
 
-    def expectedKey(index: Int): Int = {
-      if (withSizeLimit)
-        50 + index
-      else
-        index
-    }
+    def expectedKey(index: Int): Int =
+      withSizeLimit.sum + index
```

Comment:
Lol! Calling `sum` on an `Option`!
Consider `.getOrElse(0)` instead, which I believe is much more readable, not to mention close to the intent.